flow/envs/green_wave_env.py

```diff
@@ -539,12 +570,11 @@ def get_state(self):
         for edge in self.k.scenario.get_edge_list():
             ids = self.k.vehicle.get_ids_by_edge(edge)
             if len(ids) > 0:
+                # TODO(cathywu) Why is there a 5 here?
```

Comment:
I don't know...this doesn't seem right. @kjang96 do you know why there's a 5 here?